src/tfr_navigation/src/navigation_goal_manager.cpp

```diff
@@ -59,17 +64,17 @@ move_base_msgs::MoveBaseGoal NavigationGoalManager::initialize_goal() {
  */
 move_base_msgs::MoveBaseGoal NavigationGoalManager::get_updated_mining_goal(geometry_msgs::Pose msg)
 {
-    if (location_code == tfr_msgs::NavigationGoal::TO_MINING){
+    if (goal == tfr_utilities::LocationCode::MINING){
         nav_goal.target_pose.header.stamp =ros::Time::now();
```

Comment:
Formatting nitpick: add a space after the equals sign

Reply:
got it